src/components/QuizGameQuestions/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./index.css";
+
+// this function will display the fetched data to the dom.those parameters will pass in quizgame component.
+// the callback is used to check if the answer is true or not.
+// types of props ={
+//  question: string
+//  storedQuestionsAnswers string
+//  callback function
+//  userAnswer string 
+//  questionNum number
+//  totalQuestions number
+//}
+
+const QuestionCard = ({
+  question,
+  storedQuestionsAnswers,
+  callback,
+  userAnswer,
+  questionNum,
+  totalQuestions,
+}) => (
+  <div className="quizQuestionWrapper">
+    <p className="quizQuestionWrapper__numOfQuizQuestion">
+      Question: {questionNum} / {totalQuestions}
+    </p>
+    <p dangerouslySetInnerHTML={{ __html: question }} />
+    <div>
+      {storedQuestionsAnswers.map((answer) => (
+        <div
+          key={answer}
+          >
+          <button
+            className="quizQuestionWrapper__userAnswer"
+            disabled={userAnswer ? true : false}
+            value={answer}
+            onClick={callback} >
+            <span dangerouslySetInnerHTML={{ __html: answer }} />
```

Comment:
I remember reading a comment by Louis about injecting HTML in react on one of us ( can't remember which one ).  If the user gets to specify the contents of the answer itself, dangerouslySetInnerHTML should not be used as it opens ways to Cross-Site Scripting(XSS) attacks. 
I stumbled upon this article while researching this: 
https://www.pluralsight.com/guides/how-to-use-static-html-with-react

Reply:
@YamanSammani  actually it allowed here, you can read louis comment above